### DISCUSSION TITLE
How to use

### DISCUSSION THREAD

**Discussion by NullCrurivastator** [Nephelym/Breeders-Of-The-Nephelym-Save-Game-Editor#1] [General]
i am new to github stuff but everytime i tried in the past to download a program like this or have done so, they never come with anything that can open them or have a runnable program to use them. i need help using this editor cause i don't have the slightest clue as to how to use it after downloading the files...

edit: i figured it out 👍

**Comment by dafangalator** [comment]
Think you could help a brother out? I'm trying to figure it out myself

**Reply by NullCrurivastator** [reply]
Scroll all the way down and look for initial release. It should be the first or second zip download when you click it.
